core/main/src/firebolt/handlers/internal_rpc.rs

```diff
@@ -145,3 +182,22 @@ impl RippleRPCProvider<InternalImpl> for InternalProvider {
         (InternalImpl { state }).into_rpc()
     }
 }
+
+impl PolicyState {
+    pub async fn add_policy_identifier_alias(&mut self, policy: PolicyIdentifierAlias) {
+        let mut policy_identifiers_alias = self.policy_identifiers_alias.write().unwrap();
+        if policy.policy_identifier_alias.is_empty() {
```

Comment:
The usage of same member field name policy_identifier_alias  in both `policy` and `PolicyState` are confusing. Use different field names.